core/controllers/contributor_dashboard_test.py

```diff
@@ -49,19 +49,9 @@
 class ContributorDashboardPageTest(test_utils.GenericTestBase):
     """Test for showing contributor dashboard pages."""
 
-    def test_page_with_disabled_contributor_dashboard_leads_to_404(
-        self
-    ) -> None:
-        config_services.set_property(
-            'admin', 'contributor_dashboard_is_enabled', False)
-        self.get_html_response(
-            feconf.CONTRIBUTOR_DASHBOARD_URL, expected_status_int=404)
-
     def test_page_with_enabled_contributor_dashboard_loads_correctly(
```

Comment:
Can we also update the test name:
```suggestion
    def test_contributor_dashboard_page_loads_correctly(
```

Reply:
Done